langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -288,6 +401,10 @@ public isolated function padStart(string str, int len, Char padChar = " ") retur
 # Adds sufficient `padChar` characters to the end of `str` to make its length be `len`.
 # If the length of `str` is >= `len`, returns `str`.
 #
+# ```ballerina
+# "Ballerina for developers".padEnd(30, "!") ⇒ Ballerina for developers!!!!!!
```

Comment:
Can add an example without the `padChar` (defaulting) too.

Reply:
Done